include/seqan3/alignment/configuration/align_config_aligned_ends.hpp

```diff
@@ -390,7 +414,25 @@ end_gaps(ends_t && ...) -> end_gaps<std::remove_reference_t<ends_t>...>;
 
 /*!\brief The configuration for aligned sequence ends.
  * \ingroup configuration
- * \tparam end_gaps_t The type of the end gaps. Must be of type seqan3::end_gaps.
+ * \tparam end_gaps_t The type of the end gaps. Must be of type seqan3::align_cfg::end_gaps.
+ *
+ * \details
+ *
+ * This configuration element configures the aligned ends to further refine the global alignment algorithm.
+ * Particularly, the ends of the alignment can be penalised with gap costs or not. For example, the semi-global
+ * alignment does not penalise the leading and trailing gaps of one sequence while it does for the other sequence.
+ *
+ * The class is instantiated with an object of seqan3::align_cfg::end_gaps. The user can configure each of the
+ * gap specifier separately, such that it allows for maximal flexibility to configure the alignment algorithm.
+ * However, there are also predefined seqan3::align_cfg::end_gaps configurations: all_ends_free, none_ends_free,
+ * seq1_ends_free, and seq2_ends_free. Use these predefined values whenever possible.
+ *
+ * If this configuration element is not specified for the alignment algorithm it will automatically default to
+ * none_ends_free, which computes a global alignment.
```

Comment:
```suggestion
 * seqan3::align_cfg::none_ends_free, which computes a global alignment.
```

If you want to link

Reply:
No I don't because it does not work

Reply:
Literally the only changes it didn't try myself do not work 😬 
It has to be `seqan3::align_cfg::aligned_ends::none_ends_free` (I tried it)
same for the `all_ends_free` and so on

Reply:
Ok, because he thinks it is a part of the aligned_ends class. But this is ugly, so I keep it as it is in the new version. (update in a second)

Reply:
I agree, it's a way too long link